src/routes/index.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Switch } from 'react-router-dom';
+import { AuthenticaedRoute, UnauthenticaedRoute } from './protectedRoute';
+import Home from '../containers/Home';
+import Login from '../containers/Login';
+
+const RouteList = () => {
+  return (
+    <Switch>
+      <AuthenticaedRoute path="/" exact component={Home} />
```

Comment:
```suggestion
      <AuthenticaedRoute path="/" exact component={ Home } />
```
how about add some spaces here. To make consistent with another places.